### ISSUE TITLE
Download of ONNX file corrupts download

### ISSUE THREAD

**Issue by jvsoest** [MaastrichtU-BISS/FAIRmodels-backend#4]

**Comment by jvsoest** [comment]
When downloading the ONNX file from models.fairmodels.org, the downloaded file is different from the original uploaded file. The ONNX runtime cannot parse the file and fails with the following error:
```
InvalidProtobuf: [ONNXRuntimeError] : 7 : INVALID_PROTOBUF : Load model from .\Tube-feeding-1.0.0.onnx failed:Protobuf parsing failed.
```

I've compared the downloaded file and original file (using vimdiff), and although it is a binary file I can see character differences.
I've also downloaded the file from the website storage location, which shows no difference, and can be executed. Hence, the problem would lie in the download functionality, which accidentally changes the file (or maybe character set?).
